### PR TITLE
Describe vehicle parts Part 1a

### PR DESCRIPTION
**Pull request by mlangsdorf**
Part 1a in a multi-step process to enhance the vehicle construction menu, similar to "Construction Menu UI enhancements #24105".

Add descriptions for all the vehicle parts. The descriptions are intended to be functional and give some indication of why you would want to install a particular part and sometimes where you would to install it.

Part 1b will add similar descriptions to the Blazemod parts.

Parts 2+ will change the vehicle interaction screen to display the descriptions, along with some of the other data from the JSON. I'd like to at least expose epower and LIFT and JACK qualities if I can. I definitely want to indicate the location field.

I expect the descriptions will generate some discussion, so I want to get them accepted before I move on.

### CODE REVIEW

**Comment by Night-Pryanik** [comment]
Some of the vehicle parts are just variants of each other, like handle or wooden handle, so their description is identical. I wonder if adding them some flag and then describing this flag in `flags.json` would be a cleaner solution? Though this of course means much more work than simple filling the `description` field.

**Comment by mlangsdorf** [comment]
That's a good suggestion.
What I would prefer to do is get the text of descriptions worked out in this PR.
A future PR will add displaying the descriptions and parsing the flags.
After that PR is complete, then I would be ready to refactor the common text into flags in another PR.

**Comment by Night-Pryanik** [comment]
Ok then. The one last thing I want to mention is two spaces after periods inside the sentences.

File: data/json/vehicleparts/boards.json [hunk 1]
```diff
@@ -28,6 +30,7 @@
     "copy-from": "clothboard",
     "type": "vehicle_part",
     "name": "cloth quarterpanel",
+    "description": "A half-height cloth wall. Keeps zombies outside the vehiclebut allows people to see over it.",
```
Comment (RadHazard) [inline data/json/vehicleparts/boards.json]: Missing a space here

File: data/json/vehicleparts/boards.json [hunk 2]
```diff
@@ -36,6 +39,7 @@
     "copy-from": "board",
     "type": "vehicle_part",
     "name": "quarterpanel",
+    "description": "A half-height metal wall. Keeps zombies outside the vehiclebut allows people to see over it.",
```
Comment (RadHazard) [inline data/json/vehicleparts/boards.json]: Same here

File: data/json/vehicleparts/boards.json (hunk 1 above)
Comment (RadHazard) [reply data/json/vehicleparts/boards.json]: You fixed the double spacing, but I was actually talking about the typo ("vehiclebut").  I probably should have clarified after Night-Pryanik's comment.  Looks like the same typo got copy-pasta'd several times as well, including a few I missed last night

**Comment by mlangsdorf** [comment]
part 2: https://github.com/CleverRaven/Cataclysm-DDA/pull/24328

**Comment by mlangsdorf** [comment]
Better descriptions for some parts, display descriptions for installed parts #24462

**Pull request merged**